### PR TITLE
Fix for chrome on windows

### LINKED ISSUE #7: Returning "" for all cookie values
I'm using python 3.5.2 on windows 7 with chrome 54.0.2840.99 m (64-bit)

### PR DESCRIPTION
**Pull request by rkhwaja**
Fixes #7

### CODE REVIEW

**Comment by borisbabic** [comment]
I'm not really familiar with the python ecosystem on windows. Is `win32crypt` common enough on windows that we don't have to include it as a dependency for windows?

**Comment by rkhwaja** [comment]
For python 3.6, the dependency is on pywin32. I think it's already included in the ActivePython install, but I don't use it. I'm not sure how popular ActivePython is nowadays.
In fact, pywin32 is not even on pypi proper - the version on there is too old to install. I had to go to  Christoph Gohlke's page of windows wheels to get an installable version. Maybe that's an indication that people ARE using ActivePython.
In summary, I wouldn't include it in the dependency list because windows installs will probably fail and I think that it will just work anyway on ActivePython.
Maybe someone who uses ActivePython can comment...

**Comment by rkhwaja** [comment]
The code at https://github.com/Arnie97/chrome-cookies/blob/master/win32crypt.py works and it just uses ctypes. I'll probably rewrite this PR to use that - the pywin32 dependency is a pain for me too.

**Comment by rkhwaja** [comment]
There you go - no extra dependencies.

**Comment by borisbabic** [comment]
Thanks :+1: 

I've released 0.6.1 to pypi.
